src/XamlStyler.dotUltimate/src/dotnet/ReSharperPlugin.XamlStyler.dotUltimate/ReSharperPlugin.XamlStyler.dotUltimate.Rider.csproj

```diff
@@ -0,0 +1,52 @@
+﻿<Project>
+
+    <PropertyGroup>
+        <LanguageTargets>$(MSBuildToolsPath)\Microsoft.CSharp.targets</LanguageTargets>
+    </PropertyGroup>
+
+    <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+    <PropertyGroup>
+        <TargetFramework>net461</TargetFramework>
+        <AssemblyName>ReSharperPlugin.XamlStyler.dotUltimate</AssemblyName>
+        <RootNamespace>$(AssemblyName)</RootNamespace>
+        <IsPackable>false</IsPackable>
+        <DefineConstants>$(DefineConstants);RIDER</DefineConstants>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="..\..\..\..\XamlStyler\XamlStyler.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <Reference Include="PresentationCore" />
+        <PackageReference Include="JetBrains.Rider.SDK" Version="$(SdkVersion)" />
+    </ItemGroup>
+
+    <!-- TODO: uncomment for xaml icons -->
```

Comment:
Keeping these in, stems from the plugin template used (https://blog.jetbrains.com/dotnet/2019/02/14/writing-plugins-resharper-rider/). May need these in the future if custom icons for actions are needed, so kept these in.